### PR TITLE
20 update the tcplfit2 vignette to address tech review comments

### PR DESCRIPTION
**Pull request by brown-jason**

### CODE REVIEW

File: vignettes/tcplfit2-vignette.Rmd [hunk 1]
```diff
@@ -71,10 +71,9 @@ agonist activity. The data is from the mc3 table in the database `invitrodb`, wh
 for these. The plotting routine `concRespPlot` is somewhat generic, and we anticipate that users will make
 their own version of this. To run this example, use the following code ...
 
-```{r example2, fig.height = 8, fig.width = 7, eval = FALSE}
+```{r example2, fig.height = 8, fig.width = 7}
   # read in the data
-  file <- "data/mc3.RData"
-  load(file=file)
+  data("mc3")
```
[the remark this answers is not in the document]
Comment (sedavid01) [reply vignettes/tcplfit2-vignette.Rmd:77]: Resolved.

File: vignettes/tcplfit2-vignette.Rmd [hunk 2]
```diff
@@ -185,9 +184,10 @@ The input for this example comes from the ACEA_AR assay. Data from the assay com
 * clowder_uid = clowder unique id for source files
 * git_hash = hash key for pre-processing scripts
 
-```{r example4_init, fig.height = 6, fig.width = 7, message=FALSE, warning = FALSE, echo = FALSE}
-load(file = "../data/mc0.rda")
+```{r example4_init, fig.height = 6, fig.width = 7, message=FALSE, warning = FALSE}
+data("mc0")
```
[the remark this answers is not in the document]
Comment (sedavid01) [reply vignettes/tcplfit2-vignette.Rmd:190]: Resolved.

File: vignettes/tcplfit2-vignette.Rmd [hunk 3]
```diff
@@ -302,11 +307,25 @@ myfun <- function(y) {
                           )
   list(list(res)) #use list twice because data.table uses list(.) to look for values to assign to columns
 }
+```
 
+**Warning: The fitting step for the full data set, `dat`, can take 7-10 minutes to run.** Hence we will provide a subset of data for curve fitting and hitcalling. The subset data only contains records of six samples.
```
Comment (sedavid01) [inline vignettes/tcplfit2-vignette.Rmd:312]: Update this block of text to
"The following code performs dose-response modeling for all spids in the dataset.  Warning <...> Hence the code chunk following provides a subset example of data for curve fitting and hitcalling.  <...>"
The <...> keeps the rest of the text as is.

**Review by sedavid01 — COMMENTED** [review]
There is a minor comment for the text in the concentration response section, but otherwise all of this looks good.  @gracezhihuizhao, if you can go ahead and make these minor changes, then I will do one final review, then it should be good for approval.

**Review by sedavid01 — APPROVED** [review]
This looks great, thank you @gracezhihuizhao.

File: vignettes/tcplfit2-vignette.Rmd (hunk 3 above)
Comment (gracezhihuizhao) [reply vignettes/tcplfit2-vignette.Rmd]: the text has been updated

**Comment by sedavid01** [comment]
@brown-jason, changes are merged to 'master' please verify when you get the chance and if any branch clean-up is needed please either do so or provide guidance/instructions. Thanks, -Sarah

**Pull request merged**